test/integration/utils.ts

```diff
@@ -98,8 +99,14 @@ export async function testTransaction(
     response.result.engine_result_message,
   )
 
+  sequence = Number(response.result.tx_json.Sequence)
+
   // check that the transaction is on the ledger
   const signedTx = _.omit(response.result.tx_json, 'hash')
   await ledgerAccept(client)
   await verifySubmittedTransaction(client, signedTx as Transaction)
 }
+
+export function getSequence(): number {
+  return sequence
+}
```

Comment:
It might be better to return the transaction retrieved during `testTransaction` instead of having a secret listener which requires `testTransaction` to run implicitly.

Reply:
Done!